shared/actions/teams/index.js

```diff
@@ -387,21 +387,6 @@ const _getDetails = function*(action: Types.GetDetails): Saga.SagaGenerator<any,
   }
 }
 
-const _changeOpenTeamSetting = function*({
```

Comment:
We def. use this, maybe it's because we're using a constant below that it doesn't realize that?

Reply:
the constant doesn't exist, it was added back in by cjb 2 days ago (before this pr)

Reply:
oh wait no, we never actually dispatch that action, so the below is removed, and now this is removed